dist/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@ep/buffer",
+  "name": "@evidenceprime/buffer",
```

Comment:
This is about pipelines def: if we are removing this, let's remove pipeline on Azure side.